universe/marathon.json.mustache

```diff
@@ -14,38 +14,27 @@
   "container": {
     "type": "MESOS",
     "volumes": [
-
       {{#service.service_account_secret}}
       {
         "containerPath": "secrets/service-account.json",
-        "secret": "serviceCredential"
-
-      {{#storage.local_persistence.enabled}}
-      },
-      {{/storage.local_persistence.enabled}}
-      {{^storage.local_persistence.enabled}}
+        "secret": "serviceCredential",
+        {{#storage.persistence.enabled}}
```

Comment:
Remove `storage.persistence.enabled`. Persistence is always enabled with portworx volumes.
Also that config has been removed from config.json